cmd/mooshy/mooshy.go

```diff
@@ -174,6 +174,31 @@ func latestMoosh(token string) (string, error) {
 	return "", errors.New("not found")
 }
 
+func pollMagicNumber(conn net.Conn, port string, addr string, connected chan struct{}) {
```

Comment:
Wouldn't it be nicer to drop the 5 retry constraints and have this function send the magic number every `interval time.Duration` until a read from `connected` succeeds?

Reply:
Also, this function is only used once in the code, so there is no need to promote it to a function.

Reply:
The rationale behind the 5 tries is that it does not block the execution of the `example.sh` script (or any other dependency for that matter) should a delivery fail, while still giving the backdoor a fair amount of time to deploy (~30 seconds).
IMHO using a function is clearer as per the functional decomposition paradigm, but feel free to refactor it.